packages/core/src/modules/sd-jwt-vc/SdJwtVcOptions.ts

```diff
@@ -30,12 +30,31 @@ export interface SdJwtVcIssuerDid {
   didUrl: string
 }
 
+export interface SdJwtVcIssuerX5c {
+  method: 'x5c'
+  /**
+   *
+   * Array of base64-encoded certificate strings in the DER-format.
+   *
+   * The certificate containing the public key corresponding to the key used to digitally sign the JWS MUST be the first certificate.
+   */
+  chain: string[]
```

Comment:
Should we also update this to x5c like we did in sphereon's libs?

Reply:
yes I already did that locally. Pushing the changes as soon as the sphereon vp pr is merged.